cluster/ClusterSession.ts

```diff
@@ -77,54 +72,54 @@ export class SessionClusterRPC implements GraknClient.Session {
         return this._options;
     }
 
-    close(): Promise<void> {
+    async close(): Promise<void> {
```

Comment:
We shouldn't use `async` here as there's no `await`
```suggestion
    close(): Promise<void> {
```